frontend/providers/dbprovider/src/services/backend/apis/create-database.ts

```diff
@@ -4,17 +4,33 @@ import { z } from 'zod';
 import { BackupSupportedDBTypeList } from '@/constants/db';
 import { updateBackupPolicyApi } from '@/pages/api/backup/updatePolicy';
 import { KbPgClusterType } from '@/types/cluster';
-import { convertBackupFormToSpec } from '@/utils/adapt';
+import { adaptDBDetail, convertBackupFormToSpec } from '@/utils/adapt';
 import { json2Account, json2CreateCluster } from '@/utils/json2Yaml';
-
+import { DBEditType, EditType } from '@/types/db';
+import { raw2schema } from './get-database';
+const schema2Raw = (dbForm: z.Infer<typeof createDatabaseSchemas.body>): DBEditType => {
+  return {
+    dbType: dbForm.type,
```

Comment:
I'm looking at the API that was created, and I see it still includes dbForm. Is that expected?